dbms/src/Storages/DeltaMerge/File/DMFile.cpp

```diff
@@ -712,33 +712,24 @@ std::set<UInt64> DMFile::listAllInPath(
         // or you may delete some writing files
         if (options.clean_up)
         {
-            if (startsWith(name, details::FOLDER_PREFIX_WRITABLE))
+            if (startsWith(name, details::FOLDER_PREFIX_WRITABLE) || startsWith(name, details::FOLDER_PREFIX_DROPPED))
```

Comment:
> From the log "[2022/09/21 00:54:23.069 +08:00] [WARN] [DMFile.cpp:721] ["DMFile:Existing temporary dmfile, removed: /data0/db/data/t_259/stable/.tmp.dmf_6486"] [thread_id=109]", it seems there is a temporary file of the same file id with the above broken file. So this issue should be caused by the absence of clearing encryption info of old temporary files.

I am still not clear why there are a `.tmp.dmf_6486` and a `dmf_6486` sharing the same encryption info.

Reply:
These two files doesn't exist at the same time. `.tmp.dmf_6486` is a temporary file which is generated by previous run and is removed at restart.
And after some investigation into the encryption key related source code, I don't find other info about this problem and realize that this is not an urgent problem and continue to dig into it will cost a lot of time. So I decide just remove the encryption info of old file which is the right thing to do and see whether the test fail again. And even the test fails again, this can help us eliminate a wrong direction

Reply:
OK